src/System.Management.Automation/engine/hostifaces/LocalPipeline.cs

```diff
@@ -269,7 +269,7 @@ internal static int ReadRegistryInt(string policyValueName, int defaultValue)
             RegistryKey key;
             try
             {
-                key = Registry.LocalMachine.OpenSubKey(Utils.GetRegistryConfigurationPrefix());
+                key = Registry.LocalMachine.OpenSubKey("SOFTWARE\\Microsoft\\PowerShell\\1\\ShellIds");
```

Comment:
This code is in a `#if !CORECLR ... #endif` block. I leave the registry key path here only for reference, so that when we clean up this `!CORECLR` block, we know what the key is.

Reply:
ok